external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaSource.scala

```diff
@@ -458,6 +457,21 @@ private[kafka010] case class KafkaSource(
 /** Companion object for the [[KafkaSource]]. */
 private[kafka010] object KafkaSource {
 
+  val INSTRUCTION_FOR_FAIL_ON_DATA_LOSS_FALSE =
+    """
+      |There may have been some data loss because some data may have been aged out in Kafka or
+      | the topic has been deleted and is therefore unavailable for processing. If you want your
```

Comment:
nit: better grammar

Some data may have been lost because they are not available in Kafka any more; either the data was aged out by Kafka or the topic may have been deleted before all the data in the topic was processed. 

Similarly change below.